src/cephadm/cephadm

```diff
@@ -2499,6 +2536,54 @@ def command_adopt():
     else:
         raise Error('adoption of style %s not implemented' % args.style)
 
+def command_adopt_prometheus(daemon_id, uid, gid):
```

Comment:
Just thinking out loud, can we make this a `@staticmethod` of `Monitoring`?

Reply:
the script is getting rather large and it would be nice to start organizing some of the functionality into classes :+1: